recipes/ncbi-datasets-cli/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "ncbi-datasets-cli" %}
+{% set version = "11.25.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ncbi/datasets/archive/release/{{ version }}.tar.gz
+  sha256: 2c202b94bc9514c717c4f8dfc8d88da7a95e0488377b10adafc55d44e8a9901a
+
+build:
+  number: 0
+  script: python {{ RECIPE_DIR }}/build.py
+
+requirements:
+  build:
+    - bazel <4.0.0
+    - go >=1.16.4
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - datasets version
+    - dataformat version
+
+about:
+  home: https://github.com/ncbi/datasets/tree/master/pkgs/ncbi-datasets-cli
+  license: Public Domain
```

Comment:
The repo states Apache License: https://github.com/ncbi/datasets/blob/master/pkgs/ncbi-datasets-cli/LICENSE

Can you fix that? Please also use SPDX identifiers: https://spdx.org/licenses/